signer/src/main.rs

```diff
@@ -74,11 +75,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         revision = signer::GIT_COMMIT,
         arch = signer::TARGET_ARCH,
         env_abi = signer::TARGET_ENV_ABI,
+        signer_version = signer::VERSION,
         "starting the sBTC signer",
     );
 
     // Load the configuration file and/or environment variables.
     let settings = Settings::new(args.config)?;
+
+    tracing::info!(
+        signer_public_key = %PublicKey::from_private_key(&settings.signer.private_key),
```

Comment:
I think I would just add this to the above startup log properties tbh, we don't really need to know that the config has been loaded successfully because it will exit with an error if it fails.

Reply:
Having the above log before the config creation makes sense to me -- we are sure we get some logs in case there were bad issues with the config itself.

I'm fine with having this log on a different logline, but I would:
 - add a method in `SignerConfig` returning the public key, and call it here
 - change the message from `constructed signer settings` to something more tailored to the public key